apps/sasquatch/build.gradle

```diff
@@ -30,4 +30,8 @@ dependencies {
     jcenterDependencyCompile "com.microsoft.azure.mobile:mobile-center-analytics:${version}"
     jcenterDependencyCompile "com.microsoft.azure.mobile:mobile-center-crashes:${version}"
     jcenterDependencyCompile "com.microsoft.azure.mobile:mobile-center-distribute:${version}"
+
+    /* Force usage this version of support annotations to avoid conflict. */
+    androidTestCompile "com.android.support:support-annotations:${rootProject.ext.supportLibVersion}"
+    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
```

Comment:
Adding don't merge label, we need to register any new OSS usage in our internal tool.